ui/src/app/components/buttons.tsx

```diff
@@ -135,10 +134,10 @@ export const Button = ({type = 'primary', style = {}, disabled = false, ...props
   />;
 };
 
-export const MenuItem = ({icon, tooltip = '', disabled = false, children, ...props}) => {
+export const MenuItem = ({tooltip = '', disabled = false, children, ...props}) => {
   return <TooltipTrigger side='left' content={tooltip}>
     <Clickable
-      data-test-id={icon}
+      data-test-id={children}
```

Comment:
this way, it grabs the text inside the element so that the data-test-id can be created with the correct (and not duplicate like "pencil" and "pencil") id.

Reply:
is "children" a div though? or is it text?

Reply:
looks like children is technically the collection of the child's elements, so I'll use `children.innerText`